streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/SavePointServiceImpl.java

```diff
@@ -54,6 +85,24 @@ public class SavePointServiceImpl extends ServiceImpl<SavePointMapper, SavePoint
 
   @Autowired private ApplicationService applicationService;
 
+  @Autowired private ApplicationConfigService configService;
+
+  @Autowired private FlinkClusterService flinkClusterService;
+
+  @Autowired private ApplicationLogService applicationLogService;
+
+  @Autowired private FlinkRESTAPIWatcher flinkRESTAPIWatcher;
+
+  private final ExecutorService executorService =
+      new ThreadPoolExecutor(
+          Runtime.getRuntime().availableProcessors() * 5,
+          Runtime.getRuntime().availableProcessors() * 10,
+          60L,
+          TimeUnit.SECONDS,
+          new LinkedBlockingQueue<>(1024),
+          ThreadUtils.threadFactory("streampark-trigger-savepoint-executor"),
```

Comment:
```suggestion
          ThreadUtils.threadFactory("trigger-savepoint-executor"),
```

`streampark` isn't needed due to all `thread pool` belong to StreamPark 😂

Reply:
Could you address all comments? And please let me know if some comments don't make sense, thanks